scripts/prepare_heat_data.py

```diff
@@ -18,6 +18,9 @@
 #     - 2018 (commertial buildings)
 #     - 2020 (residential building)
 # Heating includes both space heating and hot water
+HEAT_DEMAND_TOTAL = 275.0e6  # ~3.3 trln Btu -> 0.25 of el demand
+COOL_DEMAND_TOTAL = 275.0e6
```

Comment:
Are these based on EIA?

Reply:
That is an estimation based on EIA statistics and NREL results for housing surveys. I have been cross-checking the overall heating/cooling demands as proportions of the overall electricity demand. Official statistics is often using BTUs for heating/cooling which is a reason to mention those `3.3 trln Btu`

Completely agree that the units and a link to the sources are needed. Also, you are right that the point above on the hard-coding applies also here. I'll think how to do it in a cleaner way